application/src/js/charts/rd-data-tools.js

```diff
@@ -160,12 +160,18 @@ function uniqueDataInColumnMaintainOrder(data, index) {
 
 
 function textToData(textData) {
-    var cleanData = textData.trim();
-    if (cleanData.search('\t') >= 0) {
-        return _.map(cleanData.split('\n'), function (line) { return line.split('\t') });
-    } else {
-        return _.map(cleanData.split('\n'), function (line) { return line.split('|') });
+    var delimiter = '\t'
+    // Super-undocumented feature that means users could enter pipe-delimited data rather than tab-delimited
```

Comment:
Do we reckon this is used..?

Reply:
No, but it seems useful for testing stuff if you don't want to have to copy tab-separated stuff from a spreadsheet.

Reply:
@TheDoubleK @samuelhwilliams it was added to make the Selenium tests possible, I believe (as sending the "tab" character to the textarea causes the focus to move, rather than tab to be inserted). There might be a different way to achieve this though.